### PR TITLE
finish project 1, group 3

### PR DESCRIPTION
**Pull request by eyezick**
Comment: Pradeep, Adonias, Isaac

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def run_tests(self):
     packages=['language_detector'],
     maintainer='rmotr.com',
     tests_require=[
-        'pytest==2.9.1',
```
Comment (stevenpmartinez) [inline setup.py:35]: Why did you remove this line from the code?

Comment (eyezick) [inline setup.py:35]: I believe Santiago edited that as it was throwing an error because the Pytest came out with a later version which they didn't expect.

File: language_detector/main.py [hunk 2]
```diff
@@ -2,10 +2,39 @@
...
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    words = text.split()
+    count_dict ={}
+    
+    def word_count(words,common_words):
```
Comment (jivewade) [inline language_detector/main.py]: I believe that this function could live outside of the function detect_languages allowing use to apply to other functions if need be.

File: language_detector/main.py [hunk 3]
```diff
@@ -2,10 +2,39 @@
...
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    words = text.split()
+    count_dict ={}
+
```
Comment (jivewade) [inline language_detector/main.py:13]: I believe that this function (word_count) could live outside of the function detect_languages allowing use to apply to other functions if need be.

File: language_detector/main.py [hunk 4]
```diff
@@ -2,10 +2,39 @@
...
+                count+=1
+        return count
+        
+    for elem in LANGUAGES:
+        """ Loops through each language and calculates the corresponding common word count. """
+        lang = elem['name']
+        com_words = elem['common_words']
+        common_count = word_count(words,com_words)
+        count_dict[lang] = common_count
+
```
Comment (jivewade) [inline language_detector/main.py:28]: Same as a previous note from me, this function (language_detector) could live outside of the function detect_languages allowing use to apply to other functions if need be.

File: language_detector/main.py [hunk 5]
```diff
@@ -2,10 +2,39 @@
...
+    def word_count(words,common_words):
+        """ Returns the common word count given the list of words in a given text. """
+        count = 0
+        for word in words:
+            if word in common_words:
+                count+=1
+        return count
+        
+    for elem in LANGUAGES:
+        """ Loops through each language and calculates the corresponding common word count. """
```
Comment (jivewade) [inline language_detector/main.py:23]: Thanks for the comments and notes. Our group approached this problem differently and I was able to learn how your team solved the problem with ease due to your comments. Good job

File: language_detector/main.py (hunk 3 above)
Comment (VinceKumar) [reply language_detector/main.py:13]: Agreed ^
